fission/src/ui/panels/configuring/MatchModeConfigPanel.tsx

```diff
@@ -23,6 +30,9 @@ export interface MatchModeConfig {
     autonomousTime: number // Optional, defaults to 15
     teleopTime: number // Optional, defaults to 135
     endgameTime: number // Optional, defaults to 20
+    ignoreRotation: boolean // Optional, defaults to true
+    maxHeight: number // Optional, defaults to infinity
+    heightPenalty: number // Optional, defaults to 2
```

Comment:
I think it would be helpful to use the TS-style docstrings to explain what these mean
```ts
/**
Optional, defaults to true

If true, the height limit will be calculated relative to the base of the robot, rather than the base of the field
*/
ignoreRotation: boolean
```